test/integration/util/util.go

```diff
@@ -182,7 +204,14 @@ func NewOcRunner() *OcRunner {
 	return &OcRunner{CommandPath: p}
 }
 
+//RunCommand executes oc command with default timeout of 3600s and returns standard output, error and exitcode.
 func (k *OcRunner) RunCommand(command string) (stdOut string, stdErr string, exitCode int) {
 	stdOut, stdErr, exitCode = runCommand(command, k.CommandPath)
 	return
 }
+
+//RunCommandWithTimeout executes oc command with timeout specified in seconds and returns standard output, error and exitcode.
```

Comment:
Generally it is expected to have space after `//`.

Reply:
@budhrg good catch, thanks